src/Tools/Microsoft.VisualStudio.Templates.AssemblyInfo.Wizard/AssemblyInfoTemplateWizard.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using EnvDTE;
+
+using Microsoft.VisualStudio.TemplateWizard;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+public partial class AssemblyInfoTemplateWizard : IWizard
+{
+    private static readonly string CSharp_Minimal_AssemblyInfoTemplate = $@"using System.Runtime.InteropServices;
+
+// In SDK-style projects the assembly information is automatically
+// generated on build.
```

Comment:
I feel like we could add more info here. This is a great opportunity to teach people this new behaviour.

For example:

> In SDK-style projects such as this one, several assembly attributes that were historically defined in this file are now automatically added during build and populated with values defined in project properties. For details of which attributes are included and how to customise this process see: https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#assemblyinfo-properties

Reply:
Any URL should probably be an aka.ms link.

Reply:
I was hoping this text would be crowd-sourced via PR, thank you for stepping up :D